# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/WebRenderAPI.h
# Commit: af34904f00c7
# Full Hash: af34904f00c78daa52ed7970f64bbdd152acd508
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-12 21:43:58
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r 94db0d98be90 -r af34904f00c7 gfx/webrender_bindings/WebRenderAPI.h
--- a/gfx/webrender_bindings/WebRenderAPI.h	Fri Jul 12 05:14:05 2019 +0000
+++ b/gfx/webrender_bindings/WebRenderAPI.h	Fri Jul 12 05:14:05 2019 +0000
@@ -635,6 +635,7 @@
   wr::PipelineId mPipelineId;
   wr::LayoutSize mContentSize;
 
+  nsTArray<wr::PipelineId> mRemotePipelineIds;
   RenderRoot mRenderRoot;
   bool mSendSubBuilderDisplayList;
 
